projects/angular/src/data/datagrid/_datagrid.clarity.scss

```diff
@@ -399,7 +399,12 @@
       &:after {
         content: '';
         width: 0.05rem;
```

Comment:
It feels right to have `width` token for the separator too. You may add it to shim along side `top`.